tests/pipeline/resources/pipeline-component-should-have-prefix/defaults.yaml

```diff
@@ -0,0 +1,14 @@
+pipeline-component:
+  prefix: "from-pipeline-component-"
+
+kubernetes-app:
+  namespace: ${NAMESPACE}
+
+kafka-connector:
+  namespace: ${NAMESPACE}
```

Comment:
what is this used for?

Reply:
you mean why is this as a env variable or why is this declared ?